app/api/groups/get_permissions_can_request_help_to.feature

```diff
@@ -34,18 +46,20 @@ Feature: Get permissions can_request_help_to for a group
     When I send a GET request to "/groups/@Class/permissions/@Class/@Item"
     Then the response code should be 200
     And the response at $.granted.can_request_help_to should be:
-      | id           | name              |
-      | @HelperGroup | Group HelperGroup |
+      | id           | name              | is_all_users_group |
+      | @HelperGroup | Group HelperGroup | false              |
 
-  Scenario: Should not return helper group when set and not visible by the current user
+  Scenario: Should return helper group without the name when set and not visible by the current user
     Given I am @Teacher
     And there is a group @HelperGroup
     And there are the following item permissions:
       | item  | group  | is_owner | can_request_help_to |
       | @Item | @Class | false    | @HelperGroup        |
     When I send a GET request to "/groups/@Class/permissions/@Class/@Item"
     Then the response code should be 200
-    And the response at $.granted.can_request_help_to should be "null"
+    And the response at $.granted.can_request_help_to should be:
+      | id           | name | is_all_users_group |
+      | @HelperGroup |      | false              |
```

Comment:
Again, this makes it difficult for me to check if this return a tuple (as requested) or just an array with one tuple (which would be more consistent with the other tables).

Reply:
fixed with `<undefined>`, see comment below

Reply:
I think you didn't get my point. My question was : how can I know reading this if you checking for a `tuple<id, name, is_all_users_group>` or an array containing one tuple<id, name, is_all_users_group> ? What would be the different in the rule?

Reply:
The difference is in the JSONPath. To retrieve an array, there is a `[*]` at the end of the JSONPath:
- `$.granted.can_request_help_to` is an object
- `$.granted_via_membership.can_request_help_to[*]` is an array

Reply:
fixed